source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -40,45 +42,72 @@ guide, consider importing the following classes in your program:
+.. code-block:: java
+   :copyable: true
+   
+   current().getString("name")
 
-Since the aggregation operations are typed, you must specify the type of each
-field you reference using a typed variation of the ``getField()``
-method, such as ``getString()`` or ``getDate()``. The following code
-shows how to reference an integer field called ``quantity``:
+To specify a value in an operation, pass it to the ``of()`` method to
+convert it to a valid type. The following code shows how to reference a
+value of ``1.0``:
 
 .. code-block:: java
    :copyable: true
+   
+   of(1.0)
+
+To create an operation, chain a method to your field or value reference.
+You can build more complex operations by chaining additional methods.
 
-   var myField = current().getInteger("quantity");
+The following example creates an operation to find patients in New
+Mexico who have visited the doctor’s office at least once. The operation
+performs the following actions:
 
-To reference a constant value, use the ``of()`` method, which converts
-primitive types to types that can be used in aggregation operations. The
-following code shows how to reference a constant of value ``1.0``:
+- Checks if the size of the ``visitDates`` array is greater than ``0``
+  by using the ``gt()`` method 
+- Checks if the ``state`` field value is “New Mexico” by using the
+  ``eq()`` method
+
+The ``and()`` method links these operations so that the pipeline stage
+matches only documents that meet both criteria.
 
 .. code-block:: java
    :copyable: true
+      
+   current()
+         .getArray("visitDates")
+         .size()
+         .gt(of(0))
+         .and(current()
+                 .getString("state")
+                 .eq(of("New Mexico")));
+
+In most cases, you need to include your operation in a method such as
+``computed()`` or ``expr()`` to process it for use in an aggregation
+pipeline stage.
 
-   var myNum = of(1.0);
+.. note::
 
-When you finish composing an aggregation operation, you can pass it to
-an aggregation builder method, such as ``match()`` or ``project()`` to
-complete an aggregation pipeline stage. To learn more about these methods,
-see :ref:`Aggregates Builders <aggregates-builders>`.
+   Methods such as ``computed()`` and ``expr()`` that take values of
+   type ``TExpression`` allow you to use your expressions in
+   aggregations. For example, the ``expr()`` method processes your
+   operation as an expression that the ``match()`` method will accept.
 
-.. note::
-   
-   While these new operations can be used within existing aggregation
-   methods that take ``TExpression`` values, methods that return
-   ``Bson`` values are not usable with this feature.
+To create a usable aggregation pipeline stage, include your expression
+in an aggregates builder method such as ``match(<expression>)`` or
+``project(<expression>)``. To learn more about these methods, see
+:ref:`Aggregates Builders <aggregates-builders>`.
```

Comment:
The project and match methods do not take `<expression>`s directly. They accept plain BSON. That BSON can be in a format that is unrelated to expressions (like match's `field: 0 or 1` format). The `$expr` operator indicates that its contents are indeed an expression.

```suggestion
To create a usable aggregation pipeline stage, include your expression
in an aggregates builder method, such as in ``match(expr(<expression>))`` or
``project(fields(computed("<field name>", <expression>)))``. To learn more about these methods, see
:ref:`Aggregates Builders <aggregates-builders>`.
```